learning-team/2020-07/2020-07-23.md

```diff
@@ -0,0 +1,105 @@
+# July 23, 2020
+
+Facilitator & Note Taker: JW
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [x] Chris Manson (CM)
+- [x] Isaac Lee (IL)
```

Comment:
I had a work meeting and couldn't attend so I think it's better that we leave this unchecked. 🙂
```suggestion
- [ ] Isaac Lee (IL)
```